### ISSUE TITLE
Assets don't compile

### ISSUE THREAD

**Issue by BjornMelgaard** [pencilcheck/ng_classify#7]
Server can't find files and trigger `couldn't find file` exeption, I investigate this by `rake assets:precompile` and find that files are saved they original name and uncompiled, example `/home/bjorn/Documents/poller/public/assets/app.coffee.ng-classify`

application is located [there](https://github.com/BjornMelgaard/poller/tree/ng_classify) on ng_classify branch

P.S.
rails (4.2.6)
sprockets (3.6.0)
sprockets-rails (3.0.4)
ng_classify (0.1.4)
railties (4.2.6)

**Comment by pencilcheck** [comment]
I don't have a lot of time nowadays, but I looked into your application briefly.

I noticed that in your compiled application.js
https://raw.githubusercontent.com/BjornMelgaard/poller/ng_classify/public/assets/application-c1945b0bb354cbd47c749bc69cf5b4babe4eace2ec8ce27308ab7d1579f3f070.js

The bottom you will see
// require app

Which I believe means that sprocket didn't put the app.coffee.ng-classify files into the pipeline.

You might want to make sure that your //= require_tree and //= require app actually works

**Comment by BjornMelgaard** [comment]
Yes, sprocket find  app.coffee if this name  without .ng-classify suffix
